docs/user/ComplianceChecking.md

```diff
@@ -0,0 +1,121 @@
+# Topology Compliance Checking  
+The Topology Compliance Checking of Winery enables to describe restrictions, constraints, and requirements for Topology Templates in form of reusable topology-based Compliance Rules.
+Furthermore, the Compliance Checker of Winery can be used to ensure that a given Topology Template is compliant to the current set of Compliance Rules
+
+## Structure of a Compliance Rule
+A Compliance Rule itself consists of two Topology Templates called
+Identifier and Required Structure.
+The Identifier is used to detect areas in a Topology Template that are subject to the rule while the
+RequiredStructure is used to verify if the rule is satisfied. It is important to note that a compliance rule is only valid if there is exactly one matching mapping from the Identifier to the RequiredStructure.
+A Topology Element 𝑒1 can be matched to a Topology Element 𝑒2 if the type of 𝑒2 is of the same type or supertype of the type of 𝑒1. Furthermore, all attributes of 𝑒1 must also be present in 𝑒2 with equivalent 𝐾𝑒𝑦 and 𝑉𝑎𝑙𝑢𝑒.
+
+An example of a valid compliance rule is shown here:
+
+![complianceRule.PNG](graphics/compliance-checking/complianceRule.PNG)
+
+The Identifier consists of a single Node Template with a property "DataType: PersonalData". As the Required Structure contains a Node Template of the same type and with a property with equivalent key and value it exists a matching mapping.
+Hence, the compliance rule is valid.
+
+## Concept of Compliance Checking
+The detection of relevant compliance areas in Topology Models is based on the detection of subgraph isomorphisms that also consider the types and attributes of the Topology Elements that form the Topology Model.
+An example of a compliance rule that is **detected** and **satisfied** for a Topology Template is shown here:
+
+![complianceRulesDetection.png](graphics/compliance-checking/complianceRuleDetection.PNG)
+
+**Detected** means: A compliance rule is detected if there is at least on matching mapping from the Identifier to the Topology Model
+
+**Satisfied** means: A compliance Rule is satisfied if each area that has been found by the detector also satisfies the Required Structure. Otherwise a rule is unsatisfied.
+ 
+
+## Tutorial
+In this section we will give you a simple example on how to use the topology compliance checking.
```

Comment:
`In the following, a simple example is given describing how to use Topology Compliance checking.`